sklearn/model_selection/_split.py

```diff
@@ -84,8 +84,8 @@ def split(self, X, y=None, groups=None):
         test : ndarray
             The testing set indices for that split.
 
-        Note
-        ----
+        Notes
+        -----
```

Comment:
these should be automatically detected by the docstring check tool. When I works as it should...